website/addons/dryad/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from . import routes, views, model
+
+
+MODELS = [
+    model.AddonDryadNodeSettings,
+    model.AddonDryadUserSettings,
+]
+USER_SETTINGS_MODEL = model.AddonDryadUserSettings
+NODE_SETTINGS_MODEL = model.AddonDryadNodeSettings
+ROUTES = [routes.page_routes]
+SHORT_NAME = 'dryad'
+FULL_NAME = 'Dryad'
+OWNERS = ['user', 'node']
+ADDED_DEFAULT = []
+ADDED_MANDATORY = []
+VIEWS = ['page']
```

Comment:
Unless you plan to have a separate page for this addon (i.e. "Files", "Wiki", "Contributors" on the project dashboard) pages should be `[]`